csunplugged/config/settings/testing.py

```diff
@@ -14,7 +22,7 @@
 # ----------------------------------------------------------------------------
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#secret-key
 # Note: This key only used for development and testing.
-SECRET_KEY = env('DJANGO_SECRET_KEY', default='CHANGEME!!!')  # noqa: F405
+SECRET_KEY = env('DJANGO_SECRET_KEY', default='l@@)w&&%&u37+sjz^lsx^+29y_333oid3ygxzucar^8o(axo*f')  # noqa: F405
```

Comment:
This is the same value used for local development, so thought I would make them consistent. The production key is still hidden.